201-vm-lb-winrm-windows/azuredeploy.json

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
Rename to 201-vm-winrm-lb-windows so it sits right next to the other winrm templates in the repo list